client-internal/src/main/java/org/terracotta/angela/client/Tsa.java

```diff
@@ -751,45 +747,46 @@ public ConfigToolExecutionResult detachNode(int stripeIndex, int serverIndex) {
   public Tsa attachAll() {
     List<List<TerracottaServer>> stripes = tsaConfigurationContext.getTopology().getStripes();
 
-    // Attach all servers in a stripe to form individual stripes
     for (List<TerracottaServer> stripe : stripes) {
       if (stripe.size() > 1) {
-        List<String> command = new ArrayList<>();
-        command.add("attach");
-        command.add("-t");
-        command.add("node");
-        command.add("-d");
-        command.add(stripe.get(0).getHostPort());
+        // Attach all servers in a stripe to form individual stripes
         for (int i = 1; i < stripe.size(); i++) {
+          List<String> command = new ArrayList<>();
+          command.add("attach");
+          command.add("-t");
+          command.add("node");
+          command.add("-d");
+          command.add(stripe.get(0).getHostPort());
           command.add("-s");
           command.add(stripe.get(i).getHostPort());
-        }
-        ConfigToolExecutionResult result = configTool(stripe.get(0)).executeCommand(command.toArray(new String[0]));
-        if (result.getExitStatus() != 0) {
-          throw new RuntimeException("ConfigTool::executeCommand with command parameters failed with: " + result);
+
+          ConfigToolExecutionResult result = configTool(stripe.get(0)).executeCommand(command.toArray(new String[0]));
+          if (result.getExitStatus() != 0) {
+            throw new RuntimeException("ConfigTool::executeCommand with command parameters failed with: " + result);
```

Comment:
there's a double ::

Reply:
Yes, that's the scope resolution operator (the kind Method references use)

Reply:
if that is linked to the method name, it would be helpful for debugging to also log the parameters that were passed

Reply:
That is happening already inside the config tool invocation :-)